instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/GarbageCollector.java

```diff
@@ -66,43 +73,52 @@ static void registerObservers(
       Function<Notification, GarbageCollectionNotificationInfo> notificationInfoExtractor) {
     Meter meter = RuntimeMetricsUtil.getMeter(openTelemetry);
 
-    LongHistogram gcTime =
+    DoubleHistogramBuilder gcDurationBuilder =
         meter
             .histogramBuilder("process.runtime.jvm.gc.duration")
             .setDescription("Duration of JVM garbage collection actions")
-            .setUnit("ms")
-            .ofLongs()
-            .build();
+            .setUnit("s");
+    setGcDurationBuckets(gcDurationBuilder);
+    DoubleHistogram gcDuration = gcDurationBuilder.build();
 
     for (GarbageCollectorMXBean gcBean : gcBeans) {
       if (!(gcBean instanceof NotificationEmitter)) {
         continue;
       }
       NotificationEmitter notificationEmitter = (NotificationEmitter) gcBean;
       notificationEmitter.addNotificationListener(
-          new GcNotificationListener(gcTime, notificationInfoExtractor), GC_FILTER, null);
+          new GcNotificationListener(gcDuration, notificationInfoExtractor), GC_FILTER, null);
+    }
+  }
+
+  private static void setGcDurationBuckets(DoubleHistogramBuilder builder) {
+    if (!(builder instanceof ExtendedDoubleHistogramBuilder)) {
+      // that shouldn't really happen
+      return;
     }
+    ((ExtendedDoubleHistogramBuilder) builder)
+        .setAdvice(advice -> advice.setExplicitBucketBoundaries(emptyList()));
```

Comment:
Is there any plan to improve this api? Would be nice not to have to jump through this.

Reply:
I'm pretty sure it's going to become a part of the regular `*HistogramBuilder` interface once the advice API stabilizes.

Reply:
😎